client/src/components/OrgBook/OrgBookData.js

```diff
@@ -0,0 +1,28 @@
+export const LANG_MAP = [
```

Comment:
WHat's the need for this? Don't we have language mapping in some language file.

Reply:
This is only to facilitate the TinyMCE editor language switching feature.   If/when the user has taken a non-default language, the editor configuration will match up (map) the selected language with the language pack found online.